src/TTextEdit.cpp

```diff
@@ -1354,6 +1372,53 @@ void TTextEdit::copySelectionToClipboardHTML()
     return;
 }
 
+void TTextEdit::searchSelectionOnline()
+{
+    // TODO this could be a separate function like QString ::getSelectedText()
+    if ((mPA.y() == mPB.y()) && (mPA.x() > mPB.x())) {
+        swap(mPA, mPB);
+    }
+    if (mPA.y() > mPB.y()) {
+        swap(mPA, mPB);
+    }
+    QString text;
+
+    for (int y = mPA.y(); y <= mPB.y() + 1; y++) {
+        if (y >= static_cast<int>(mpBuffer->buffer.size())) {
+            mSelectedRegion = QRegion(0, 0, 0, 0);
+            QString url = text.trimmed();
+            url.prepend(mpHost->mSearchEngine.second);
+            QDesktopServices::openUrl(QUrl(url));
+            forceUpdate();
+            break;
```

Comment:
I think you might as well use a `return` here since that has the same effect and it'll be more obvious that you intend to quit right away.